.buildkite/pipeline.yml

```diff
@@ -0,0 +1,33 @@
+steps:
+  - command: .buildkite/scripts/run_autoformat.py
+  - wait
+  - command: .buildkite/scripts/run_job.py internal_model
+  - command: .buildkite/scripts/run_job.py display
+  - command: .buildkite/scripts/run_job.py elasticsearch
+  - command: .buildkite/scripts/run_job.py big_messaging
+  - command: .buildkite/scripts/run_job.py ingestor_common
+  - command: .buildkite/scripts/run_job.py transformer_common
+  - wait
+  - command: .buildkite/scripts/run_job.py api
+  - command: .buildkite/scripts/run_job.py reindex_worker
+  - command: .buildkite/scripts/run_job.py ingestor_works
+  - command: .buildkite/scripts/run_job.py ingestor_images
+  - command: .buildkite/scripts/run_job.py id_minter
+  - command: .buildkite/scripts/run_job.py matcher
+  - command: .buildkite/scripts/run_job.py merger
+  - command: .buildkite/scripts/run_job.py recorder
+  - command: .buildkite/scripts/run_job.py transformer_miro
+  - command: .buildkite/scripts/run_job.py transformer_sierra
+  - command: .buildkite/scripts/run_job.py transformer_mets
+  - command: .buildkite/scripts/run_job.py transformer_calm
+  - command: .buildkite/scripts/run_job.py sierra_reader
+  - command: .buildkite/scripts/run_job.py sierra_bib_merger
+  - command: .buildkite/scripts/run_job.py sierra_item_merger
+  - command: .buildkite/scripts/run_job.py sierra_items_to_dynamo
+  - command: .buildkite/scripts/run_job.py mets_adapter
+  - command: .buildkite/scripts/run_job.py calm_adapter
+  - command: .buildkite/scripts/run_job.py inference_manager
+  - command: .buildkite/scripts/run_job.py feature_inferrer --changes-in pipeline/inferrer/feature_inferrer
+  - command: .buildkite/scripts/run_job.py snapshot_generator
+  - wait
+  - command: .buildkite/scripts/complete_build.py
```

Comment:
Is there anything in particular I need to do to set up `relation_embedder` beyond adding a `run_job` command here?

Reply:
The `run_job.py` script does the same thing as the one for `Travis` essentially so you still need to ensure there are `make` tasks available.